tests/test_full_smrf.py

```diff
@@ -8,28 +8,30 @@
 Tests for an entire smrf run. The SMRF integration run!
 """
 
-import unittest
 import shutil
-import smrf
-import os
-from os.path import abspath, join, dirname
-from smrf.framework.model_framework import can_i_run_smrf, run_smrf
-from inicheck.tools import get_user_config, check_config, cast_all_variables
-from copy import deepcopy
+import unittest
+from os.path import abspath, dirname, isdir, join
+
 import numpy as np
 from netCDF4 import Dataset
 
-def compare_image(v_name, gold_dir, test_dir, msg):
+import smrf
+from smrf.framework.model_framework import run_smrf
+
+
+def compare_image(v_name, gold_dir, test_dir):
```

Comment:
Is this used anywhere else?

Reply:
run_smrf or compare_image?

run _smrf is used in the setUpClass function to build the data
compare_image is used in the loop to test all the variables.

Reply:
`compare_image`

Reply:
compare_image is only used in test_full_smrf, so if someone does use it anywhere else they will need to assertTrue on it.